dissect/target/plugins/os/windows/regf/shimcache.py

```diff
@@ -15,6 +15,8 @@
 ShimcacheRecord = TargetRecordDescriptor(
     "windows/shimcache",
     [
+        ("string", "name"),
+        ("varint", "entry_index"),
```

Comment:
```suggestion
        ("varint", "index"),
```
I think "entry" is already implied and thus redundant.